app/views/providers/training_providers_courses.csv.erb

```diff
@@ -0,0 +1,9 @@
+<%
```

Comment:
I think this could live in the course model via `to_csv` to match by above suggestion.

Reply:
I dont' understand why this file is here tbh....